invenio_app_rdm/theme/templates/invenio_app_rdm/frontpage.html

```diff
@@ -1,17 +1,178 @@
 {#
   Copyright (C) 2019 CERN.
-  Copyright (C) 2019 Northwestern University, Galter Health Sciences Library & Learning Center.
+  Copyright (C) 2019 Northwestern University.
 
   Invenio App RDM is free software; you can redistribute it and/or modify it
   under the terms of the MIT License; see LICENSE file for more details.
 #}
 {%- extends "invenio_theme/frontpage.html" %}
 
+{%- block head_links %}
+{{ super() }}
+<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600|Oswald|Source+Code+Pro:400&amp;display=swap" rel="stylesheet">
+{%- endblock head_links %}
+
+
 {%- block page_body %}
-<div class="container marketing">
-  <div class="row">
-    <div class="col-lg-12">
-    <h1 class="text-center">Welcome to Invenio App RDM.</h1>
+
+<section class="invenio-rdm-project-section section-content-white-bg">
+  <div class="container">
+    <span class="section-title">You've installed InvenioRDM successfully!</span>
```

Comment:
Comment:

"You've successfully installed InvenioRDM!"

Reply:
Agreed! You (and I) prefer it because, readers naturally pay more attention to the first and last word of a sentence.

Reply:
:+1: